modules/plugins/utility/motion/flash/flash.nix

```diff
@@ -0,0 +1,36 @@
+{lib, ...}: let
+  inherit (lib.options) mkEnableOption mkOption;
+  inherit (lib.types) nullOr str;
+in {
+  options.vim.utility.motion.flash = {
+    enable = mkEnableOption "flash.nvim plugin";
```

Comment:
Would be nice to describe what the plugin actually does here. For most plugin the format is

```nix
mkEnableOption "something something [flash.nvim]";

``` or similar while accounting for the fact that `mkEnableOption` prepends "Whether to enable" to the description.